src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/MediaTransportControls_themeresources.xaml

```diff
@@ -3,6 +3,8 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:not_ios="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:ios="http://platform.uno/ios"
```

Comment:
This should technically also be an issue on Android. It may be trying to render a native android app bar inside of the MediaTransportControls

Reply:
Good point. Let me run the sample on an Android device

Reply:
Surprisingly, there is no issue with Android. This is running the latest stable **Uno.SDK 5.2.175**

| Material | Fluent |
| :--: | :--: |
|![android-uno-mpe-controls-material](https://github.com/unoplatform/uno/assets/1900897/5b61d71e-cc6a-4e67-be9a-6719de56ba67)|![android-uno-mpe-controls-fluent](https://github.com/unoplatform/uno/assets/1900897/c80ce3ea-a462-4401-9eda-9ac1b3e7e674)|